test/mix/tasks/guardian.db.gen.migration_test.exs

```diff
@@ -0,0 +1,21 @@
+defmodule Mix.Tasks.Guardian.Db.Gen.MigrationTest do
```

Comment:
When do I need to call this task?

Reply:
This is a test mate. It's testing that the migration runs correctly and doesn't bomb out.